### PR TITLE
[PVR] Series Recordings - Coverity / Code Cleanup

### PR DESCRIPTION
**Pull request by ksooo**
Followup to #7079
- Coverity fixes
- Late code review feedback
- Small fixes

@Jalle19, @opdenkamp mind taking a look

### CODE REVIEW

File: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp [hunk 1]
```diff
@@ -955,9 +925,9 @@ void CGUIDialogPVRTimerSettings::AddTypeDependentEnableCondition(CSetting *setti
   AddCondition(setting, id, TypeReadOnlyCondition, SettingDependencyTypeEnable, SETTING_TMR_TYPE);
 }
 
-// static
 bool CGUIDialogPVRTimerSettings::TypeReadOnlyCondition(const std::string &condition, const std::string &value, const CSetting *setting, void *data)
 {
+  bool bRet(true);
```
Comment (Jalle19) [inline xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp]: This doesn't seem to fit in?

**Comment by Jalle19** [comment]
Except for that comment it looks fine

File: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp (hunk 1 above)
Comment (ksooo) [inline xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp]: Yeah. Thanks. No idea how this one slipped in. Will ofc be removed.

**Comment by ksooo** [comment]
jenkins build this please

**Comment by ksooo** [comment]
jenkins build this please

File: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp (hunk 1 above)
Comment (xhaggi) [inline xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp]: you forgot to remove this unused var

Comment (ksooo) [inline xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp]: Ooops. Will do with my next PR.

Comment (ksooo) [inline xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp]: https://github.com/ksooo/xbmc/commit/b4177e04912bdfc09514560e283f1a31a108a691#diff-81ee6b50ffdcedb2e947a807469bf195L930

**Pull request merged**